less/mixins.less

```diff
@@ -1 +1,30 @@
-// Mixins in here
\ No newline at end of file
+.g-button(@width, @height, @bgcolor, @font-color) {
+   
+    .border-radius(20px);
+    .center();
+    width:@width;
+    height:@height;
+    background:@bgcolor;
+    color:@font-color;
+    font-size:2rem;
+    cursor:pointer;
+    margin: 20px 0;
+
+    &:hover {
+        color:@bgcolor;
+        background:@font-color;
+        border:1px solid @bgcolor;
+    }
```

Comment:
excellent use here of a nested, parametric mixin to allow a great deal of control with very little effort involved after writing the initial mixin